html/janus.js

```diff
@@ -86,6 +86,20 @@ var defaultExtension = {
 	}
 };
 
+function stopAllTracks (stream) {
```

Comment:
Had to move this up, otherwise, it would not have been able to be called from `Janus.init()`.
Not sure where to put this, if something else is desired please let me know.

Reply:
You're right, I didn't think about that.
Moving the function declaration at the top sounds not good to me.

IMHO it's better to revert the for loop stopping the tracks in the `Janus.init()` and declare `stopAllTracks` in the scope of the `Janus` function (like it was before your last edit). Then re-use `stopAllTracks` whenever its declaration is in scope.